FHIR/Resource+Operation.swift

```diff
@@ -19,24 +19,11 @@ public extension Resource {
 	
 	
 	// MARK: - Search
-	
-	/**
-	Perform a search against the receiver's type's compartment.
-	
-	UNFINISHED.
-	*/
-	public func search(_ query: Any) -> FHIRSearch {
-		if let _ = self.id {
-			NSLog("UNFINISHED, must add '_id' reference to search expression")
-			//return FHIRSearch(subject: "_id", reference: myID, type: type(of: self))
-		}
-		return FHIRSearch(type: type(of: self), query: query)
-	}
```

Comment:
This function caused a build error when I removed `public` from the other `search()` function. Since the "unfinished" function was not used anywhere, I deleted it.